deploy/helm/sumologic/templates/setup/setup-custom-configmap.yaml

```diff
@@ -9,6 +8,7 @@ metadata:
     app: {{ template "sumologic.labels.app.setup.configmap-custom" . }}
     {{- include "sumologic.labels.common" . | nindent 4 }}
 data:
+{{- if .Values.sumologic.setup.additionalFiles }}
```

Comment:
Why this change? It should be a separate PR if we need it IMO.

Reply:
Just merged master. Just be good now